src/raw/cgroupv2.go

```diff
@@ -81,7 +81,7 @@ func (cg *CgroupsV2Fetcher) Fetch(c types.ContainerJSON) (Metrics, error) {
 		log.Error("couldn't get cpu count: %v", err)
 	}
 
-	if stats.Memory, err = cg.memory(metrics); err != nil {
+	if stats.Memory, err = cg.memory(metrics, c); err != nil {
```

Comment:
I like short names... but `c` is really short 😄 

(I know you inherited it, but if it is not complex I would change it to something more meaningful now that we are passing it down)

Reply:
That's right, let's improve it a little bit 🙂

Reply:
Changed: [c91a728](https://github.com/newrelic/nri-docker/pull/248/commits/c91a728bfcf483f8639c96664f9eb3d05739af22)